JBBCode/Tokenizer.php

```diff
@@ -23,26 +23,20 @@ class Tokenizer
      */
     public function __construct($str)
     {
-        $strStart = 0;
         $strLen = strlen($str);
-        for ($index = 0; $index < $strLen; ++$index) {
-            if (']' == $str[$index] || '[' == $str[$index]) {
-                /* Are there characters in the buffer from a previous string? */
-                if ($strStart < $index) {
-                    $this->tokens[] = substr($str, $strStart, $index - $strStart);
-                    $strStart = $index;
-                }
+        $position = 0;
 
-                /* Add the [ or ] to the tokens array. */
-                $this->tokens[] = $str[$index];
-                $strStart = $index+1;
+        while($position < $strLen) {
```

Comment:
This isn't used universally in the project, but I'd like to try to stick to the [PSR-2 style guide](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md), which includes spaces between control structure keywords and the opening parenthesis. So,

```php
while($position < $strLen) {
```
should be
```php
while ($position < $strLen) {
```
Same for the `if` down below.